src/main/webapp/js/instructorFeedbackEdit.js

```diff
@@ -1022,19 +1019,19 @@ function addMsqOption(questionNumber) {
             parseInt($('#' + FEEDBACK_QUESTION_NUMBEROFCHOICECREATED + idSuffix).val());
         
     $("<div id=\"msqOptionRow-" + curNumberOfChoiceCreated + idSuffix + "\">"
-        +   "<div class=\"input-group\">"
-        +       "<span class=\"input-group-addon\">"
-        +          "<input type=\"checkbox\" disabled>"
-        +       "</span>"
-        +       "<input type=\"text\" name=\"" + FEEDBACK_QUESTION_MSQCHOICE + "-" + curNumberOfChoiceCreated + "\" "
-        +               "id=\"" + FEEDBACK_QUESTION_MSQCHOICE + "-" + curNumberOfChoiceCreated + idSuffix + "\" class=\"form-control msqOptionTextBox\">"
-        +       "<span class=\"input-group-btn\">"
-        +           "<button type=\"button\" class=\"btn btn-default removeOptionLink\" id=\"msqRemoveOptionLink\" "
-        +                   "onclick=\"removeMsqOption(" + curNumberOfChoiceCreated + "," + questionNumber + ")\" tabindex=\"-1\">"
-        +               "<span class=\"glyphicon glyphicon-remove\"></span>"
-        +           "</button>"
-        +       "</span>"
-        +   "</div>"
+          + "<div class=\"input-group\">"
+              + "<span class=\"input-group-addon\">"
+                 + "<input type=\"checkbox\" disabled>"
+              + "</span>"
+              + "<input type=\"text\" name=\"" + FEEDBACK_QUESTION_MSQCHOICE + "-" + curNumberOfChoiceCreated + "\" "
+                      + "id=\"" + FEEDBACK_QUESTION_MSQCHOICE + "-" + curNumberOfChoiceCreated + idSuffix + "\" class=\"form-control msqOptionTextBox\">"
+              + "<span class=\"input-group-btn\">"
+                  + "<button type=\"button\" class=\"btn btn-default removeOptionLink\" id=\"msqRemoveOptionLink\" "
+                          + "onclick=\"removeMsqOption(" + curNumberOfChoiceCreated + "," + questionNumber + ")\" tabindex=\"-1\">"
+                      + "<span class=\"glyphicon glyphicon-remove\"></span>"
+                  + "</button>"
+              + "</span>"
+          + "</div>"
```

Comment:
The trade-off is you're allowing `var a =  *1` (where `*` is a whitespace), or even `var a = ****1;`. Alternatively we can eslint-disable this section (and many others).